setup.py

```diff
@@ -27,12 +27,22 @@
     sys.exit(1)
 print("This is backports.lzma version %s" % __version__)
 
+lzmalib = '%slzma'%('lib' if sys.platform == 'win32' else '')
+
+class build_ext_subclass(build_ext):
+    def build_extensions(self):
+        c = self.compiler.compiler_type
+        if c == "mingw32" and sys.maxsize <= 2**32:
+           for e in self.extensions:
+               e.extra_compile_args = ["-mstackrealign"]
+        build_ext.build_extensions(self)
+
 packages = ["backports", "backports.lzma"]
 prefix = sys.prefix
 home = os.path.expanduser("~")
-extens = [Extension('backports/lzma/_lzma',
+extens = [Extension('backports.lzma._lzma',
                     ['backports/lzma/_lzmamodule.c'],
```

Comment:
Would have figured you would need `\` on Windows. Maybe using `os.path.join` would work?

Reply:
According to the documentation of distutils, 'name' is:
```
    the full name of the extension, including any packages - ie. not a
    filename or pathname, but Python dotted name
```

Reply:
Oh, you were referring to the source argument, no, that's not really required, I think. Docs say:
```
list of source filenames, relative to the distribution root (where the setup script lives), in Unix form (slash- separated) for portability. 
```

Reply:
The Appveyor CI ran the build with this PR. See https://ci.appveyor.com/project/peterjc/backports-lzma/build/1.0.23

Reply:
Does it also work with the original Linux style slashes?

Reply:
Yes, it is portable.